apps/src/music/progress/MusicValidator.ts

```diff
@@ -30,16 +30,35 @@ export default class MusicValidator extends Validator {
   constructor(
     private readonly getIsPlaying: () => boolean,
     private readonly getPlaybackEvents: () => PlaybackEvent[],
+    private readonly getExemplarPlaybackEvents: () => PlaybackEvent[],
     private readonly getValidationTimeout: () => number,
     private readonly player: MusicPlayer,
     private readonly getPlayingTriggers: () => PlayingTrigger[],
+    private readonly exemplarSettings: ExemplarSettings,
     private readonly conditionsChecker: ConditionsChecker = new ConditionsChecker(
       Object.values(MusicConditions).map(condition => condition.name)
     )
   ) {
     super();
   }
 
+  shouldValidateWithExemplar(): boolean {
+    return this.exemplarSettings?.validationEnabled ?? false;
+  }
+
+  getExemplarValidationResults(): {satisfied: boolean; message: string} {
+    if (!this.shouldValidateWithExemplar()) {
+      throw new Error('Attempted to validated with exemplar when disabled.');
```

Comment:
nit: `validate`.  Also, what exactly is disabled in this case?  The exemplar?

Reply:
Updated: `Exemplar validation is disabled; cannot retrieve exemplar validation results.`